css-brochure-form/styles.css

```diff
@@ -0,0 +1,48 @@
+body h2 {
+  text-align: center;
+  color: rgb(230, 77, 21);
+  font-family: sans-serif;
+  text-transform: uppercase;
+}
+
+#form-auto {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  height: 90vh;
+}
+label {
+  display: block;
+}
+input,
+select {
+  display: block;
+  margin: 10px;
+  padding: 10px;
+  border: 0.1rem solid grey;
+  border-radius: 0.5rem;
+  box-sizing: border-box;
+  width: 500px;
+}
+
+button[type='submit'] {
+  margin: 10px;
+  padding: 10px 22px;
+  border-radius: 1rem;
+  background: #e97917;
+  border: 0;
+  color: #fff;
+  position: relative;
+  left: 210px;
+}
+
+/* font changing styles */
+
+label[for='CEMAIL'],
+label[for='name'],
+label[for='hire'],
+label[for='monthSelector'] {
+  font-family: 'Brandon Grotesque', sans-serif;
+}
```

Comment:
Since you want this same font for all of your `label`s, I would just add the `font-family` property to your `label` selector on line 16. No need to target each one of these individually, since you want all of them to have the same font